client/human/HumanClientFSM.cpp

```diff
@@ -591,6 +591,8 @@ WaitingForTurnData::~WaitingForTurnData()
 boost::statechart::result WaitingForTurnData::react(const SaveGameDataRequest& msg) {
     if (TRACE_EXECUTION) DebugLogger() << "(HumanClientFSM) WaitingForTurnData.SaveGameDataRequest";
     DebugLogger() << "Sending Save Game Data to Server";
+    Client().GetClientUI()->GetMessageWnd()->HandleGameStatusUpdate(
+        boost::io::str(FlexibleFormat(UserString("SERVER_SAVE_INITIATE_ACK"))) + "\n");
```

Comment:
no need for a string substitution if you're not inserting anything into the looked up string...